valve/addons/amxmodx/scripting/agmodx_llhl.sma

```diff
@@ -0,0 +1,439 @@
+    gCvarBlockNameChangeInMatch = create_cvar("sv_ag_block_namechange_inmatch", "1");
+
+    // Make respawn delay consistent in all fps values (already implemented by BugfixedHL)
+    gCvarRespawnFix = get_cvar_pointer("mp_respawn_fix");
+
+    hook_cvar_change(gCvarFpsCheckInterval, "CvarAgFpsCheckIntervalHook");
+    hook_cvar_change(gCvarRespawnFix, "CvarAgRespawnFixHook");
+}
+
+public agmodx_post_config() {
+    if (get_pcvar_num(gCvarCheckSoundFiles)) {
+        // Sound file consistency
+        for (new i; i < sizeof gConsistencySoundFiles; i++) {
+            force_unmodified(force_exactfile, {0,0,0}, {0,0,0}, gConsistencySoundFiles[i]);
+        }
+    }
+}
+
+public plugin_init() {
+    register_plugin(PLUGIN, VERSION, AUTHOR);
+
+    register_dictionary("agmodx_llhl.txt");
+
+    register_forward(FM_StartFrame, "FwStartFrame");
+
+    register_clcmd("say /unstuck", "CmdUnstuck");
+
+    RegisterHam(Ham_Spawn, "player", "Ham_Player_Spawn_Pre", 0);
+    RegisterHam(Ham_Spawn, "player", "Ham_Player_Spawn_Post", 1);
+    RegisterHam(Ham_Killed, "player", "Ham_Player_Killed_Post", 1);
+    RegisterHam(Ham_Use, "game_end", "Ham_Game_End");
+
+    register_message(SVC_INTERMISSION, "FwMsgIntermission");
+
+    register_forward(FM_CmdStart, "fmCmdStart_Pre");
+    register_forward(FM_SetModel, "FwSetModel");
+    register_forward(FM_ClientUserInfoChanged, "FwClientUserInfoChanged");
+
+    // Add vote for mp_respawn_fix (Only in LLHL gamemode)
+    ag_vote_add("mp_respawn_fix", "OnVoteRespawnFix");
+}
+
+public plugin_cfg() {
+    set_task(get_pcvar_float(gCvarFpsCheckInterval), "taskMeasureMeanFPS", TASK_FPSLIMITER, .flags = "b");
+    hook_cvar_change(get_cvar_pointer("sv_ag_match_running"), "CvarMatchRunningHook");
+}
+
+public inconsistent_file(id, const filename[], reason[64]) {
+    new name[32], authid[32];
+    get_user_name(id, name, charsmax(name));
+    get_user_authid(id, authid, charsmax(authid));
+    client_print(0, print_chat, "%l", "LLHL_FILECONSISTENCY_MSG", name, authid, filename);
+    server_cmd("kick #%d ^"%l^"", get_user_userid(id), "LLHL_FILECONSISTENCY_KICK", filename);
+    return PLUGIN_HANDLED;
+}
+
+public client_connect(id) {
+    gIsAlive[id] = false;
+    gMeanFPS[id] = gLastProbeFPS[id] = 0.0;
+    gNumDetections[id] = 0;
+}
+
+public client_disconnected(id) {
+    gIsAlive[id] = false;
+}
+
+public Ham_Player_Spawn_Pre(id) {
+    gIsAlive[id] = false;
+    return HAM_IGNORED;
+}
+
+public Ham_Player_Spawn_Post(id) {
+    gIsAlive[id] = bool:is_user_alive(id);
+    gProbeCount[id] = gProbeSum[id] = 0;
+    return HAM_IGNORED;
+}
+
+public Ham_Player_Killed_Post(id) {
+    gIsAlive[id] = false;
+    return HAM_IGNORED;
+}
+
+// Create task before intermission because we can't run tasks when in intermission mode
+public FwMsgIntermission() {
+    gActualServerFPS = gServerFPS;
+    client_cmd(0, "stop;wait;wait;+showscores");
+    set_task(0.1, "taskShowVEngine", TASK_SHOWVENGINE, .flags = "b");    
```

Comment:
Not sure why you use a task to show this message

Reply:
If I send any kind of message in the hook the server crashes, so I use a task for that